specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2021-03-01-preview/nova.json

```diff
@@ -0,0 +1,1143 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Cosmos DB - Managed Cassandra",
```

Comment:
Consider renaming the json file to something like ManagedCassandra,json

Reply:
Done.